### PR TITLE
feat: added tooltips for navigation bar (#193)

### PR DESCRIPTION
**Pull request by boyney123**
**Description**

Following #193, I added some tooltips to the navigation bar.

Let me know what you think @mcturco @magicmatatjahu

### CODE REVIEW

File: src/components/Sidebar.tsx [hunk 1]
```diff
@@ -116,6 +117,7 @@ export const Sidebar: React.FunctionComponent<SidebarProps> = () => {
       <div className="flex flex-col">
         <SettingsModal />
       </div>
+      <ReactTooltip effect='solid' backgroundColor='#374151' arrowColor='#374151' className="text-xs"  />
```
Comment (magicmatatjahu) [inline src/components/Sidebar.tsx:120]: So if I wanna add tooltip in another part of Studio, e.g. in Editor, then I should copy that component and use `data-tip` attributes? 🤔  I think that component that wraps given component and add tooltip will be better, something like:

```tsx
<Tooltip content={...}>
  //...some component
</Tooltip>
```

is there any another tooltip component in the OpenSource?

**Review by magicmatatjahu — CHANGES_REQUESTED** [review]
I have a problem where exactly tooltip is showed, check this:

image

image

as you can see if I move the cursor in the nav item, but not at the icon I don't see any tooltip. It should be integrated not with icon, but with whole nav item :)

**Comment by mcturco** [comment]
Thanks @boyney123 for adding this!! 🎉   However, I think if we made the background of the labels fully opaque it would look a lot cleaner. I think it might be a little distracting to see things peeking through behind the labels as the text size is already rather small. What do you think?

**Comment by mcturco** [comment]
@boyney123 @magicmatatjahu 

Hey guys! Was cleaning out my inbox and realized that this PR has yet to be merged. I think this is still a great feature that we should consider revisiting and merging. I think all we should do is make the tool tips fully opaque instead of transparent and integrate them with the entire button and not just the icon. Then we should definitely ship it! What do you guys think?

**Comment by magicmatatjahu** [comment]
Tooltips added in the https://github.com/asyncapi/studio/pull/434 PR 

@boyney123 I hope that you don't have a problem that I'm closing this PR :)

**Pull request closed without merging**